lib/ios/AirGoogleMaps/AIRGoogleMapPolyline.m

```diff
@@ -49,6 +52,39 @@ -(void)setStrokeWidth:(double)strokeWidth
   _strokeWidth = strokeWidth;
   _polyline.strokeWidth = strokeWidth;
 }
+-(void)setLineDashPattern:(NSArray <NSNumber *> *)lineDashPattern {
+  _lineDashPattern = lineDashPattern;
+}
+-(void)updateLineDashPattern:(double)scale {
+  
+  // scale = 1.0 / mapView.projection.pointsForMeters(1, atCoordinate: mapView.camera.target)
+//  NSArray *lengths = @[@25, @20];
+  NSMutableArray *scalePatterns = [[NSMutableArray alloc] init];
+  for(int i=0; i< _lineDashPattern.count; i++){
+    [scalePatterns addObject:@(_lineDashPattern[i].intValue * scale)];
+  }
+  _scale = scale;
+  NSArray *styles = @[[GMSStrokeStyle solidColor: _strokeColor],
+                      [GMSStrokeStyle solidColor:[UIColor clearColor]]];
+  _polyline.spans = GMSStyleSpans(_polyline.path, styles, scalePatterns, kGMSLengthRhumb);
+}
+
+-(void)updateLineDash:(double)scale {
+
+  NSMutableArray *scalePatterns = [[NSMutableArray alloc] init];
+  for(int i=0; i< _lineDashPattern.count; i++){
+    [scalePatterns addObject:@(_lineDashPattern[i].intValue * scale)];
+  }
+  NSArray *styles = @[[GMSStrokeStyle solidColor: _strokeColor],
```

Comment:
Can't find documentation, but isn't it `styles` and `scalePatterns` should be same size?

Reply:
nah, styles are the colors one being the line color other transparent to create the "dash"